using_packages/using_profiles.rst

```diff
@@ -0,0 +1,121 @@
+.. _using_profiles:
+
+Using profiles
+--------------
+
+So far we have used the default settings stored in ``~/.conan/profiles/default`` and defined as command line arguments.
+
+However, configurations can be large, settings can be very different, and we might want to switch easily between different configurations
+with different settings, options, etc.. The best way to this is using profiles.
+
+A profile file contains a predefined set of ``settings``, ``options``, ``environment variables``, and ``build_requires`` and has this
+structure:
+
+.. code-block:: text
+
+    [settings]
+    setting=value
+
+    [options]
+    MyLib:shared=True
+
+    [env]
+    env_var=value
+
+    [build_requires]
+    Tool1/0.1@user/channel
+    Tool2/0.1@user/channel, Tool3/0.1@user/channel
+    *: Tool4/0.1@user/channel
+
+Options allow definition with wildcards, to apply same option value to many packages:
+
+.. code-block:: text
+
+    [options]
+    *:shared=True
+
+They would contain the desired configuration, for example:
+
+.. code-block:: text
+   :caption: myprofile
```

Comment:
In this first example, I wouldn't introduce options, only the env and settings stuff and I would name it "clang_3.5" instead of myprofile, to be reused later

Reply:
Explain where to save the file